pages/caritonaranjo/style.css

```diff
@@ -0,0 +1,111 @@
+body{
+    background-color: black;
+    font-family: 'Tenali Ramakrishna', sans-serif;
+    color: cornsilk;
+   }
+#carousel{
```

Comment:
is it an id? we should be using only classes

Reply:
is an ID, but for calling the button's actions, not for style.